packages/entryPoints/index.ts

```diff
@@ -303,6 +303,18 @@ export async function startPreview() {
     wsScene = `${location.protocol === 'https:' ? 'wss' : 'ws'}://${document.location.host}/?scene`
   }
 
+  getPreviewSceneId().then(async (sceneData) => {
+    if (sceneData.sceneId) {
+      const { unityInterface } = await ensureUnityInterface()
+      unityInterface.SetKernelConfiguration({
+        debugConfig: {
+          sceneDebugPanelTargetSceneId: sceneData.sceneId!,
+          sceneLimitsWarningSceneId: sceneData.sceneId!
+        }
+      })
+    }
```

Comment:
I just saw where the bug of last week should have been resolved, lines https://github.com/decentraland/kernel/pull/65/files#diff-1200c56879b0f91125578151317e698e17f82d7d386495ca503416aa8fa45013R291-R293

 ```suggestion
      if(!queryString.has('position')){
           const position = pickWorldSpawnpoint(scene)
           i.Teleport(position)
           teleportObservable.notifyObservers(position.position)
      }
    }
```